e2e/providerconfig-aws-e2e-test/Makefile

```diff
@@ -0,0 +1,106 @@
+# Project Setup
+PROJECT_NAME := providerconfig-aws-e2e-test
+PROJECT_REPO := github.com/upbound/provider-upjet-aws
```

Comment:
This URL returns a 404. I thought it was either `github.com/crossplane-contrib/provider-upjet-aws` or `github.com/upbound/provider-aws`. But maybe we're using a different convention here?

Reply:
It has been some time since this PR was initiated, so probably it switched orgs after I opened the PR. Will fix this